components/dialog/dialog.js

```diff
@@ -151,9 +156,11 @@ class Dialog extends LocalizeCoreElement(AsyncContainerMixin(DialogMixin(LitElem
 			'd2l-footer-no-content': !this._hasFooterContent
 		};
 
+		if (!this._textId) this._textId = getUniqueId();
+
 		const content = html`
 			${loading}
-			<div style=${styleMap(slotStyles)}><slot></slot></div>
+			<div id="${this._textId}" aria-describe-content="${this.ariaDescribeContent}" style=${styleMap(slotStyles)}><slot></slot></div>
```

Comment:
I don't think you need the `aria-describe-content="${this.ariaDescribeContent}"` here.

Reply:
you are right! I will take that out :)